src/app/issuer/components/badgeclass-detail/badgeclass-detail.component.ts

```diff
@@ -258,12 +372,8 @@ export class BadgeClassDetailComponent extends BaseAuthenticatedRoutableComponen
 							icon: 'lucidePencil',
 						},
 						{
-							title: 'Badge.copyThisIssuer',
-							action: () => {
-								this.router.navigate(['/issuer/issuers', this.issuer.slug, 'badges', 'create'], {
-									state: { copybadgeid: this.badgeSlug },
-								});
-							},
+							title: 'Badge.copy',
```

Comment:
To fulfill whats in the bug description, shouldn't we make this `copy_permissions.includes('others') ? 'General.copy' : 'Badge.copyThisIssuer'` ?